shared/chat/inbox/row/filter-big-team-channel/index.js

```diff
@@ -19,17 +21,41 @@ type Props = {
   onSelectConversation: () => void,
 }
 
-class FilterBigTeamChannel extends PureComponent<Props> {
+type State = {
+  isHovered: boolean,
+}
+
+const HoverBox = isMobile
+  ? Box
+  : glamorous(Box)({
```

Comment:
note we don't need this anymore with this pr (https://github.com/keybase/client/pull/13870). There are generated classNames for hover so you can just add `className='.hover_color_blueGrey2'`